modules/deploy/pages/deployment-option/cloud/security/cloud-availability.adoc

```diff
@@ -3,10 +3,10 @@
 :page-cloud: true
 
 Redpanda Cloud supports the deployment of Redpanda clusters in single or multiple
-availability zones (AZs), spanning at most three AZs. Nodes are evenly distributed
+glossterm:availability zone (AZ)[,AZs], spanning at most three AZs. Brokers are evenly distributed
```

Comment:
```suggestion
glossterm:availability zones[,AZs], spanning at most three AZs. Brokers are evenly distributed
```

Reply:
The term name is availability zones: https://raw.githubusercontent.com/redpanda-data/docs/shared/modules/terms/partials/az.adoc

It should be singular really, but to get this to work, you must match the defined term name.